search_config.module

```diff
@@ -3,7 +3,7 @@
 /*
  * Helper function for the installation of the module
  */
-namespace  Drupal\search_config;
+namespace  Drupal\module;
```

Comment:
Remove the namespace line completely from here. let's not add any namespace in .module file. keep the functions in .module file procedural